crates/apub/src/api/search.rs

```diff
@@ -57,21 +57,23 @@ pub async fn search(
   let creator_id = data.creator_id;
   let local_user = local_user_view.as_ref().map(|l| &l.local_user);
 
+  let mut posts_query = PostQuery {
+    sort: (sort),
+    listing_type: (listing_type),
+    community_id: (community_id),
+    creator_id: (creator_id),
+    local_user,
+    search_term: (Some(q.clone())),
+    page: (page),
+    limit: (limit),
+    ..Default::default()
+  };
+
```

Comment:
If you dont mind you could also do the same thing for CommentQuery, CommunityQuery and PersonQuery, as each of them is written twice now for different search types. Would be some nice cleanup but not necessary for this PR.

Reply:
Ok, will include that as well.